### PR TITLE
Adds IssuedCurrencyClient::enableRipplingForTrustLine

### PR DESCRIPTION
**Pull request by tedkalaw**
## High Level Overview of Change
Adds `IssuedCurrencyClient::enableRipplingForTrustLine`, which removes the `noRipple` flag

### Context of Change
Opposite of https://github.com/xpring-eng/Xpring-JS/pull/871

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Before / After
`IssuedCurrencyClient` now has `clearNoRipple`

## Test Plan
CI, my test

### CODE REVIEW

File: src/XRP/issued-currency-client.ts [hunk 1]
```diff
@@ -531,6 +531,32 @@ export default class IssuedCurrencyClient {
...
+   * @param trustLinePeerAccount The X-Address of the account involved in the trust line to re-enable rippling.
+   * @param currencyName The currency of the trust line to re-enable rippling.
+   * @amount The maximum amount of debt to allow on this trust line.
+   * @param wallet The wallet re-enabling rippling on the trust line.
+   */
+  public async clearNoRipple(
+    trustLinePeerAccount: string,
+    currencyName: string,
+    amount: string,
+    wallet: Wallet,
```
[the remark this answers is not in the document]
Comment (mvadari) [reply src/XRP/issued-currency-client.ts:549]: I think if we want an additional `flags` method, we should just provide a method for changing a bunch of flags at once, with some enum that determines the flag options

File: src/XRP/issued-currency-client.ts [hunk 2]
```diff
@@ -531,6 +531,32 @@ export default class IssuedCurrencyClient {
...
+   * Note that the trust line's limit is set to 0.
+   *
+   * @see https://xrpl.org/freezes.html#enabling-or-disabling-individual-freeze
+   *
+   * @param trustLinePeerAccount The X-Address of the account involved in the trust line to re-enable rippling.
+   * @param currencyName The currency of the trust line to re-enable rippling.
+   * @amount The maximum amount of debt to allow on this trust line.
+   * @param wallet The wallet re-enabling rippling on the trust line.
+   */
+  public async clearNoRipple(
```
Comment (mvadari) [inline src/XRP/issued-currency-client.ts:545]: I think this name is a bit confusing - I would suggest something like `enableRipple`, but that's the DefaultRipple flag (maybe that should be changed to `enableDefaultRippling`?)

File: src/XRP/issued-currency-client.ts [hunk 3]
```diff
@@ -531,6 +531,32 @@ export default class IssuedCurrencyClient {
     )
   }
 
+  /**
+   * Re-enables rippling on the trust line between this account (issuing account) and another account.
+   * Note that the trust line's limit is set to 0.
+   *
+   * @see https://xrpl.org/freezes.html#enabling-or-disabling-individual-freeze
```
Comment (mvadari) [inline src/XRP/issued-currency-client.ts:538]: Is this the right link?

File: src/XRP/issued-currency-client.ts [hunk 4]
```diff
@@ -531,6 +531,32 @@ export default class IssuedCurrencyClient {
...
+   *
+   * @param trustLinePeerAccount The X-Address of the account involved in the trust line to re-enable rippling.
+   * @param currencyName The currency of the trust line to re-enable rippling.
+   * @amount The maximum amount of debt to allow on this trust line.
```
Comment (amiecorso) [inline src/XRP/issued-currency-client.ts:542]:
```suggestion
   * @param amount The maximum amount of debt to allow on this trust line.
```

File: src/XRP/issued-currency-client.ts [hunk 5]
```diff
@@ -531,6 +531,32 @@ export default class IssuedCurrencyClient {
     )
   }
 
+  /**
+   * Re-enables rippling on the trust line between this account (issuing account) and another account.
+   * Note that the trust line's limit is set to 0.
```
Comment (amiecorso) [inline src/XRP/issued-currency-client.ts:536]: again, is this true??  I don't think it is

Comment (tedkalaw) [reply src/XRP/issued-currency-client.ts]: it's not, removed

File: src/XRP/issued-currency-client.ts (hunk 3 above)
Comment (tedkalaw) [reply src/XRP/issued-currency-client.ts]: yeup

File: src/XRP/issued-currency-client.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (tedkalaw) [reply src/XRP/issued-currency-client.ts]: sgtm. let's revisit later

File: src/XRP/issued-currency-client.ts (hunk 2 above)
Comment (tedkalaw) [reply src/XRP/issued-currency-client.ts]: i tried `enableTrustLineRippling` - let me know what you think. i'll go back and change `setNoRipple` to `disableTrustLineRippling` if we like this naming

**Review by mvadari — APPROVED** [review]

File: src/XRP/issued-currency-client.ts [hunk 6]
```diff
@@ -564,6 +564,31 @@ export default class IssuedCurrencyClient {
     )
   }
 
+  /**
+   * Re-enables rippling on the trust line between this account (issuing account) and another account.
+   *
+   * @see https://xrpl.org/rippling.html#enabling-disabling-no-ripple
+   *
+   * @param trustLinePeerAccount The X-Address of the account involved in the trust line to re-enable rippling.
+   * @param currencyName The currency of the trust line to re-enable rippling.
```
Comment (amiecorso) [inline src/XRP/issued-currency-client.ts:573]:
```suggestion
   * @param currencyName The currency of the trust line being re-enabled to ripple.
```

File: src/XRP/issued-currency-client.ts [hunk 7]
```diff
@@ -564,6 +564,31 @@ export default class IssuedCurrencyClient {
     )
   }
 
+  /**
+   * Re-enables rippling on the trust line between this account (issuing account) and another account.
+   *
+   * @see https://xrpl.org/rippling.html#enabling-disabling-no-ripple
+   *
+   * @param trustLinePeerAccount The X-Address of the account involved in the trust line to re-enable rippling.
```
Comment (amiecorso) [inline src/XRP/issued-currency-client.ts:572]:
```suggestion
   * @param trustLinePeerAccount The X-Address of the account involved in the trust line being re-enabled to ripple.
```

File: src/XRP/issued-currency-client.ts (hunk 6 above)
Comment (amiecorso) [reply src/XRP/issued-currency-client.ts:573]: lol sorry to be a nit, I just want our docs to at least be grammatically correct

**Review by amiecorso — APPROVED** [review]

File: src/XRP/issued-currency-client.ts (hunk 6 above)
Comment (tedkalaw) [reply src/XRP/issued-currency-client.ts:573]: hmm that seems fair i suppose

**Pull request merged**